mergeSort1.rb

```diff
@@ -1,5 +1,4 @@
 @array = [1, 2, 7, 98, 45, 5, 1, 4, 56, 23]
-@merged = []
```

Comment:
got rid of @merged and replaced it with ary on line 21.
I did this because as you merge you want to keep creating new arrays until you get the final sorted array
```[1] [12] [5] [7]```
so if we merge 1 and 12 first then we get
```[1,12]``
if this was @merged then when we handle
```[5] [7]``
we would add 5 first but adding it to @merged would create
```[1,12,5]`` and then
```[1,12,5,7]```
instead each time we merge we want to start with a fresh empty array so that we can merge them index by index
```[1,5,7,12]```